data/json/items/tools.json

```
@@ -9160,6 +9160,82 @@
     "looks_like": "frame_wood_light",
     "flags": [ "NONCONDUCTIVE" ]
   },
+  {
+    "id": "sword_xiphos",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": "xiphos",
+    "name_plural": "xiphoses",
+    "description": "A bronze sword of ancient Greek origin, wielded as a sidearm to the dory spear.",
+    "weight": 800,
+    "volume": 8,
+    "price": 12000,
+    "to_hit": 2,
+    "bashing": 8,
+    "cutting": 20,
+    "material": "bronze",
+    "symbol": "!",
+    "color": "yellow",
+    "techniques": [ "WBLOCK_2" ],
+    "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 8 ] ],
+    "flags": [ "SHEATH_SWORD", "DURABLE_MELEE" ]
+  },
+  {
+    "id": "khopesh",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": "khopesh",
+    "name_plural": "khopeshes",
+    "description": "This ancient bronze weapon features a curved, sickle-like blade sharpened on the outside edge.  Associated with the New Kingdom period of ancient Egypt, it was designed mainly to hack through the light armor common to the region.",
+    "weight": 750,
+    "volume": 7,
+    "price": 12000,
+    "to_hit": 1,
+    "bashing": 7,
+    "cutting": 19,
+    "material": "bronze",
+    "symbol": "/",
+    "color": "yellow",
+    "techniques": [ "WBLOCK_2", "DEF_DISARM" ],
+    "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 8 ] ],
+    "flags": [ "DURABLE_MELEE" ]
+  },
+  {
+    "id": "dao",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": "dao",
+    "name_plural": "dao",
+    "description": "An ancient Chinese sword consisting of a curved blade and a guard with a cupped design.  Existing since the Shang dynasty, this one is made of bronze.  One of the four major weapons in folklore, alongside the jian sword, qiang spear, and gun staff.",
+    "weight": 850,
+    "volume": 7,
+    "price": 12500,
+    "to_hit": 2,
+    "bashing": 6,
+    "cutting": 24,
```

A bit out of line for a small bronze sword.
```suggestion
    "cutting": 18,
```